### ISSUE TITLE
[issue]: NiceHash IMG Boots to Console Screen (unexpected)

### ISSUE THREAD

**Issue by asheroto** [ventoy/Ventoy#1702]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.76

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

32GB

### Disk Manufacturer

Samsung

### Image file checksum (if applicable)

No.

### Image file download link (if applicable)

https://files.nicehash.com/nhminer/nhos/nhos-1.2.11/image/nhos-1.2.11.img.gz

### What happened?

IMG file for booting is inside the archive linked.

[NiceHash.com](https://www.nicehash.com/) has a release of "NiceHash OS" which is a small bootable OS based on Tiny Core Linux. It's a small OS solely for mining crypto.

See [user guide here](https://www.nicehash.com/guide/nicehash-os-user-guide) for details.

It is supposed to boot into something that looks like this...

![good](https://user-images.githubusercontent.com/49938263/176545053-4153f9f4-d894-488c-bf96-46dc27ec1ab3.jpg)

But when using Ventoy, whether BIOS or UEFI mode, whether regular or memory mode (F1), it just brings you to a console screen like this...

![bad](https://user-images.githubusercontent.com/49938263/176545076-9ceff0b1-06f4-4014-ab5a-1d867860cf0d.jpg)

Using the same image direct to a flash drive it boots as expected, but not when used with Ventoy.

Thank you. :-)

**Comment by H34R7L3s** [comment]
Did you fix this issue?
I have the same problem :/

**Comment by hassan0ibrahim** [comment]
Did you fix this issue?

**Comment by asheroto** [comment]
No, I never fixed the issue. Some glitch with Ventoy booting. Ended up just making it's own dedicated flash drive for this one.
